app/server/src/errors/handleError.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from "express";
+import { uid } from "uid";
+import { reqWithError } from "../logging";
+import { BeebopError } from "./beebopError";
+
+function sendError(response, error, status = 500) {
```

Comment:
Any need for a default here as it's always called with arg in place?

Reply:
Also, any reason why this function does not have type annotations?

Reply:
Went through a couple of iterations, I think there was a use of the default value at one point. Will remove, and add some type annotations too.